contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6376,34 +6360,37 @@ bbf_ExecDropStmt(DropStmt *stmt)
 			schema_oid = get_object_namespace(&address);
 			if (OidIsValid(schema_oid))
 				schema_name = get_namespace_name(schema_oid);
-			if (schema_name != NULL)
-				logicalschema = get_logical_schema_name(schema_name, true);
-
-			if (schema_name && major_name)
+			if (sql_dialect == SQL_DIALECT_TSQL)
 			{
-				if (stmt->removeType == OBJECT_TABLE)
-				{
-					type = ExtendedPropertyTypeNames[EXTENDED_PROPERTY_TABLE];
-					delete_extended_property(db_id, type, schema_name,
-											 major_name, NULL);
-					type = ExtendedPropertyTypeNames[EXTENDED_PROPERTY_TABLE_COLUMN];
-					delete_extended_property(db_id, type, schema_name,
-											 major_name, NULL);
-				}
-				else if (stmt->removeType == OBJECT_VIEW)
-				{
-					type = ExtendedPropertyTypeNames[EXTENDED_PROPERTY_VIEW];
-					delete_extended_property(db_id, type, schema_name,
-											 major_name, NULL);
-				}
-				else if (stmt->removeType == OBJECT_SEQUENCE)
+				if (schema_name != NULL)
+					logicalschema = get_logical_schema_name(schema_name, true);
```

Comment:
logicalscheam can be null here? is it possible?

Reply:
If schema_name is not NULL, logicalschema should not be NULL. 
We can set the missingOk flag to false to throw an error in case of any exception.

Reply:
Changed the default value of missingOk flag to `false` in the latest commit.

Reply:
@Deepesh125 `logical_schema` being `NULL` case is already being handled inside `clean_up_bbf_schema_permissions`, setting it to `false` in the function call is breaking alot of existing test cases.